content/shared/influxdb3-admin/tokens/admin/list.md

```diff
@@ -8,8 +8,45 @@ data and resources in your InfluxDB 3 instance.
 > Token metadata includes the hashed token string.
 > InfluxDB 3 does not store the raw token string.
 
-In the following examples, replace {{% code-placeholder-key %}}`AUTH_TOKEN`{{% /code-placeholder-key %}} with your InfluxDB {{% token-link "admin" %}}
-{{% show-in "enterprise" %}} or a token with read permission on the `_internal` system database`{{% /show-in %}}.
+## Prerequisites
+
+Before running the following examples, obtain a valid InfluxDB {{% token-link "admin" %}}.  
+
+{{% show-in "enterprise" %}}Alternatively, use a token with read access to the `_internal` system database.{{% /show-in %}}
+
+Use one of the following methods to provide your token:
+
+{{% code-placeholders "your-token" %}}
+
+{{< code-tabs-wrapper >}}
+
+{{% code-tabs %}}
+[CLI](#cli-auth)
+[HTTP API](#http-api-auth)
+{{% /code-tabs %}}
+
+{{% code-tab-content %}}
+```bash
+# Set your InfluxDB 3 token as an environment variable
+export INFLUXDB3_AUTH_TOKEN=your-token
+```
+{{% /code-tab-content %}}
+
+{{% code-tab-content %}}
+
+```bash
+# Add your token to the HTTP Authorization header
+--header "Authorization: Bearer your-token"
+```
+
+{{% /code-tab-content %}}
+
+{{< /code-tabs-wrapper >}}
+
+{{% /code-placeholders %}}
+
+Replace `your-token` with your actual InfluxDB 3 token.
```

Comment:
Rather than including this content here on this specific page, I think you should add a section about different methods of providing a token on the "Manage tokens" landing page. Then send users to that section if they want more information.

On this page specifically, use an "important" block to call out the need for specific token permissions.

```suggestion
> [!Important]
> #### Required permissions
>
> Listing admin tokens requires a valid InfluxDB {{% token-link "admin" %}}{{% show-in "enterprise" %}} or a token with read access to the `_internal` system database{{% /show-in %}}.
> For more information about providing a token, see <LINK_TO_NEW_TOKEN_CONTENT>.
```